default-application-catalog/cert-manager-app.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apps.kubermatic.k8c.io/v1
+kind: ApplicationDefinition
+metadata:
+  name: cert-manager
+spec:
+  description: cert-manager is a Kubernetes addon to automate the management and issuance of TLS certificates from various issuing sources.
+  method: helm
+  versions:
+  - template:
+      source:
+        helm:
+          chartName: cert-manager
+          chartVersion: v1.12.3
+          url: https://charts.jetstack.io
+    version: v1.12.3
+  # During the applicationdefinition creation time if the defaultValues is updated, then it is render for override during the application creation process from UI.
```

Comment:
I find this sentence a bit confusing. But I am having a tough time making a suggestion at the moment, because I have trouble understanding the sentence. What is the the point you would like to get across with this sentence?

Reply:
Ignore the comment which I added for my reference and missed to remove. Removed now from all the definitions. :)